cloudlink/cloudlink.py

```diff
@@ -18,7 +18,7 @@ def __init__(self):
 
     def server(self, logs=False):
         # Initialize Cloudlink server
-        from .server import server
+        from server import server
```

Comment:
this got fixed in a later commit, github being dumb and not showing it

Reply:
This breaks importing as a module, which is the intended functionality of CL.

`>>> from cloudlink import Cloudlink`
`>>> cl = Cloudlink()`
`>>> server = cl.server(logs=True)`
`Traceback (most recent call last):`
`  File "<stdin>", line 1, in <module>`
`  File "C:\Users\mikie\Documents\GitHub\package\cloudlink\cloudlink.py", line 21, in server`
`    from server import server`
`ModuleNotFoundError: No module named 'server'`